Missing angular-confirm.min.js by installing via npm

Hi, James. You developed this wonderful module, thank you so much, dude. But we are confused by some strange things:
1) there's a line in `package.json`:
```json
"main": "angular-confirm.min.js"
```
2) but! in `.gitignore` you specified same file - `angular-confirm.min.js`

By installing this module via npm we have a problem: module does not contain `angular-confirm.min.js` because of `.npmignore` which totaly matches with `.gitignore`. Could you please update the package in npm providing an appropriate `.npmignore` (which wouldn't contain `angular-confirm.min.js`).

We've got this error because of missing `angular-confirm.min.js`:
```
Module not found: Error: Cannot resolve module 'angular-confirm'
```

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "angular-confirm",
   "version": "1.2.5",
   "description": "Angular Confirm Modal",
-  "main": "angular-confirm.min.js",
+  "main": "angular-confirm.js",
   "scripts": {
     "test": "gulp test-unit"
   },

